### ISSUE TITLE
Version 4.0 missing on Bionic

### ISSUE THREAD

**Issue by rvaser** [egor-tensin/setup-clang#1]
Dear Egor,
I am unable to setup Clang 4.0 with the following snippet:
```yaml
name: Setup Clang
uses: egor-tensin/setup-clang@v1
with:
  version: 4.0
  platform: x64
```
I get the errror:
```bash
E: The repository 'http://apt.llvm.org/bionic llvm-toolchain-bionic-4 Release' does not have a Release file.
```
The version is missing from https://apt.llvm.org/bionic/dists/.

Best regards,
Robert

P.S. Full workflow is at https://github.com/rvaser/test/blob/master/.github/workflows/test.yml.

**Comment by egor-tensin** [comment]
@rvaser This is a peculiarity of YAML syntax. `4.0` gets parsed as just `4`, you need to put it in quotes to prevent that. I fixed in my fork of your repository in commit https://github.com/egor-tensin/test/commit/c5cab8ddb6b1f1df7146497a69796546e4ef1e44. The resulting run is here: https://github.com/egor-tensin/test/actions/runs/1010820169.

I'll fix that in the action itself a bit later. Thanks you for reporting this.

**Comment by rvaser** [comment]
Thanks a lot for the quick reply. Any reason why 5.0 works without quotes (https://github.com/rvaser/test/tree/4cd4f22efb4a6ed115f023b4055fc4f0f2990211)?

**Comment by egor-tensin** [comment]
@rvaser Yeah, I actually predicted that this issue would occur, and mapped `5` to the canonical `5.0`, and `6` to `6.0` inside the action. I missed mapping `4` to `4.0` because for some reason I only looked at the version available on apt.llvm.org, omitting those in the offical repositories.

Anyway, it was fixed in commit f553c1487dc2df78cb75a3c66a1f96321d2a7d91 (== v1.1.1 == v1).

**Comment by rvaser** [comment]
Thanks a lot! :)
